.github/workflows/build.yml

```diff
@@ -0,0 +1,43 @@
+name: build
+on:
+    workflow_dispatch:
+    push:
+jobs:
+    build:
+        name: Build and upload to riffraff
+        runs-on: ubuntu-latest
+        # Allow GitHub to request an OIDC JWT ID token, to use with aws-actions/configure-aws-credentials
+        permissions:
+            id-token: write
+            contents: read
+        steps:
+            - name: Checkout the repository
+              uses: actions/checkout@v4
+
+            - name: Install Node
+              uses: actions/setup-node@v3
+              with:
+                  node-version-file: '.nvmrc'
+
+            - name: Install dependencies, build & run tests
+              run: |
+                npm install -g yarn
```

Comment:
Yarn comes [pre-installed in the environment](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#package-management), so we should be able to drop this line.